### PR TITLE
Extend PSA: improvements

### PR DESCRIPTION
**Pull request by ToLToL**
- Changed policy names and titles.
- Add an example to exclude fields outside container objects with labels.

### CODE REVIEW

File: proposals/extend_pod_security_admission.md [hunk 1]
```diff
@@ -72,114 +71,149 @@ For example, if a user wants to apply Restricted PSS to the selected namespaces
...
-* Host Namespaces: `spec.hostNetwork,spec.hostPID,spec.hostIPC`
-* Sysctls: `spec.securityContext.sysctls[*].name`
-* volumes: `spec.volumes`
-* Non-root groups: `spec.securityContext.supplementalGroups[*], spec.securityContext.fsGroup`
+- Host Namespaces: `spec.hostNetwork,spec.hostPID,spec.hostIPC`
+- Sysctls: `spec.securityContext.sysctls[*].name`
+- volumes: `spec.volumes`
+- Non-root groups: `spec.securityContext.supplementalGroups[*], spec.securityContext.fsGroup`
 
+For these pod level checks, we can use labels to select them.
```
Comment (JimBugwadia) [inline proposals/extend_pod_security_admission.md:213]: Do we have a sample policy for this?

File: proposals/extend_pod_security_admission.md [hunk 2]
```diff
@@ -72,114 +71,149 @@ For example, if a user wants to apply Restricted PSS to the selected namespaces
...
+    rule: check-for-labels
+    properties:
+      controlName: Capabilities
+      podSecurity: spec.containers[*].securityContext.capabilities.add
+    scored: true
+    status: fail
 ```
 
-### Exceptions for container security context
+### Fields outside container object
```
Comment (JimBugwadia) [inline proposals/extend_pod_security_admission.md:206]: Still trying to think of a better title here....

Maybe "Additional pod security controls"? And an sentence to explain:

Here are the pod security controls that are not declared at the container-level:

**Review by JimBugwadia — COMMENTED** [review]
thanks @ToLToL! Some minor comments...

File: proposals/extend_pod_security_admission.md (hunk 1 above)
Comment (ToLToL) [reply proposals/extend_pod_security_admission.md:213]: Yes [here](https://github.com/kyverno/KDP/pull/27/files#diff-6445fcf14a55d9c90d6e0b83d196721b466908fdfc41cb653d515994d3ad098eR151),

We should move this part just before the sample policy.

**Review by JimBugwadia — APPROVED** [review]

File: proposals/extend_pod_security_admission.md [hunk 3]
```diff
@@ -7,179 +7,226 @@
...
+        profile: restricted
+        # version must be a valid Kubernetes minor version,
+        # or `latest`.
+        version: v1.24
+        exclude:
+          # controlName is the Control defined in PSS
+          # +required
+          - controlName: Volumes
+            labels:
+              env: prod
```
Comment (realshuting) [inline proposals/extend_pod_security_admission.md:199]: The label should be under match/exclude block.

**Review by realshuting — CHANGES_REQUESTED** [review]

File: proposals/extend_pod_security_admission.md (hunk 3 above)
Comment (ToLToL) [reply proposals/extend_pod_security_admission.md]: fixed

File: proposals/extend_pod_security_admission.md [hunk 4]
```diff
@@ -72,114 +71,149 @@ For example, if a user wants to apply Restricted PSS to the selected namespaces
...
+      Rule check-for-labels failed at path
+      /metadata/labels/app.kubernetes.io/name/'
+    policy: require-labels
+    resources:
+      - apiVersion: v1
+        kind: Pod
+        name: nginx
+        namespace: default
+        uid: 13fcb726-5597-4fcd-a708-ce8b558bd484
+    rule: check-for-labels
```
[the remark this answers is not in the document]
Comment (ToLToL) [reply proposals/extend_pod_security_admission.md]: done

File: proposals/extend_pod_security_admission.md [hunk 5]
```diff
@@ -72,114 +71,149 @@ For example, if a user wants to apply Restricted PSS to the selected namespaces
...
-  properties:
-    controlName: Capabilities
-    podSecurity: spec.containers[*].securityContext.capabilities.add
-  scored: true
-  status: fail
+  - message: |-
+      'validation error: The label `app.kubernetes.io/name` is required. 
+      Rule check-for-labels failed at path
+      /metadata/labels/app.kubernetes.io/name/'
+    policy: require-labels
```
[the remark this answers is not in the document]
Comment (ToLToL) [reply proposals/extend_pod_security_admission.md]: done

**Review by realshuting — APPROVED** [review]

**Pull request merged**